src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -230,6 +230,33 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
         }
 
+        /// <summary>
+        /// Gets an &lt;input&gt; elements "type" attribute value for the given <paramref name="modelExplorer"/>.
```

Comment:
In the codez -- https://github.com/aspnet/Mvc/pull/5104/files/97a49f9044430e633d4f17675d621bd32b65dce5#diff-ed3977cf971ec6121e035dd0f173df3eR256